Formula/inspircd.rb

```diff
@@ -1,8 +1,8 @@
 class Inspircd < Formula
   desc "Modular C++ Internet Relay Chat daemon"
   homepage "https://www.inspircd.org/"
-  url "https://github.com/inspircd/inspircd/archive/v3.7.0.tar.gz"
-  sha256 "e77df17ed7f774fa0927cfce4412c2b6a3e4d5a9a8acab9753685c2aef22ea3b"
+  url "https://github.com/inspircd/inspircd/archive/v3.8.0.tar.gz"
+  sha256 "2f847c6a091332faa7a647291f92ff3ed571b17967c85198d34c816c40e47e04"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-only"
```

https://github.com/inspircd/inspircd/blob/insp3/src/channels.cpp
```
 * This file is part of InspIRCd.  InspIRCd is free software: you can
 * redistribute it and/or modify it under the terms of the GNU General Public
 * License as published by the Free Software Foundation, version 2.
```